azure/templates/tests.yml

```diff
@@ -0,0 +1,48 @@
+parameters:
+  - name: service_name
+    type: string
+
+steps:
+  - task: DockerInstaller@0
+    displayName: 'Install Docker 17.09.0-ce'
+
+
+  - bash: |
+     pip install virtualenv
+     virtualenv test_env
+     source ./test_env/bin/activate
+     pip install -r requirements.txt
+    workingDirectory: $(Pipeline.Workspace)/s/${{ parameters.service_name }}/$(SERVICE_ARTIFACT_NAME)/api_tests
+    displayName: Setup integration tests for pytest
+
+  - template: "azure/components/aws-assume-role.yml@common"
+    parameters:
+     role: "auto-ops"
+     profile: "apm_ptl"
+
+  - template: "azure/components/get-aws-secrets-and-ssm-params.yml@common"
+    parameters:
+     secret_file_ids:
+      - ptl/app-credentials/jwt_testing/non-prod/JWT_TESTING_PRIVATE_KEY
+      - ptl/app-credentials/jwt_testing/non-prod/ID_TOKEN_TESTING_PRIVATE_KEY
+      - ptl/app-credentials/jwt_testing/non-prod/ID_TOKEN_NHS_LOGIN_PRIVATE_KEY
+     config_ids: []
+     secret_ids: []
+
+  - bash: |
+     source ./test_env/bin/activate
+
+     pytest --service_name $(FULLY_QUALIFIED_SERVICE_NAME) --apigee_env $(APIGEE_ENVIRONMENT) \
+     --client_id eAMbFALgZFdnds2w5Z1u78AGCcJ0FtxQ \
```

Comment:
`client_id` is not secret. I can put this as env var however, in the pipeline it should be hardcoded since, I know platforms doesn't store them as secrets. Because it'll be harcoded in the pipeline anyway I decided to hardcode it here as well to make life easier for devs.